src/smokeTest/java/uk/gov/hmcts/reform/ccd/document/am/SmokeTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.ccd.document.am;
```

Comment:
package  name needs adjustment.

Reply:
Build is failing. Not able to update this comment at the moment.
Will update this change later.